### PR TITLE
TA integration section rework

### PR DESCRIPTION
**Pull request by karensawrey**
The Integrations section is now split into many pages. Initial commit, many website mechanics bugs are to be expected.

### CODE REVIEW

File: pages/test_analytics/importing_json.md.erb [hunk 1]
```diff
@@ -239,4 +239,23 @@ The JSON format is not yet documented, but the following example is what is prod
 
 ## Environment variables
 
-<%= render_markdown 'test-analytics/_env_variables_ta' %>
```
Comment (plaindocs) [inline pages/test_analytics/importing_json.md.erb]: This would be **without** the trailing underscore. :/

Comment (plaindocs) [reply pages/test_analytics/importing_json.md.erb]: I guess it's so Rails doesn't think it is a file it needs to care about otherwise.

Comment (karensawrey) [reply pages/test_analytics/importing_json.md.erb]: Yeah, I've been meaning to delete it before committing, but forgot to, @plaindocs

File: pages/test_analytics/importing_junit_xml.md.erb [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Importing JUnit XML
+
+<div class="Docs__wip-note">
+<p class="Docs__wip-heading">Test Analytics is an invite-only beta feature. Join the <a href="https://buildkite.com/test-analytics">waitlist</a>.</p>
+</div>
+
+{:toc}
+
+By default, the beta release ships with [RSpec collector](/docs/test-analytics/ruby-collectors#rspec-collector). However, you you can also upload test results by importing [JUnit XML](/docs/test-analytics/importing-junit-xml) or [JSON](/docs/test-analytics/importing-json).
```
Comment (plaindocs) [inline pages/test_analytics/importing_junit_xml.md.erb:9]:
```suggestion
By default, the beta release ships with [RSpec collector](/docs/test-analytics/ruby-collectors#rspec-collector). However, you you can also upload test results by importing JUnit XML or [JSON](/docs/test-analytics/importing-json).
```

**Review by plaindocs — APPROVED** [review]
This does what we discussed. 

image


I've pushed up a redirect from the old page, and made one tiny link suggestion. 

FAO @olyism her is what we have for now. Let's merge this.

**Comment by karensawrey** [comment]
Thanks for the thumbs up, @plaindocs! I wonder how this didn't run into a head-on collision with https://github.com/buildkite/docs/pull/1480, but I'm glad it didn't. Merging once the tests have given me the green light.

**Comment by plaindocs** [comment]
I sneakily fixed the merge for you. 😎

**Comment by karensawrey** [comment]
@plaindocs I suspected you would :D Thanks!

**Pull request merged**